scripts/events/egg_hunt_egg-stravaganza.lua

```diff
@@ -780,6 +780,23 @@ for bonusWord, rewardItem in pairs(settings.BONUS_WORDS) do
     })
 end
 
+-----------------------------------
+-- HELM event handler
+-----------------------------------
+xi.events.eggHunt.helmResult = function(player, itemID)
+    if
+        itemID ~= 0 and
+        xi.events.eggHunt.enabledCheck() and
+        player:getCharVar(settings.VAR.DAILY_HELM) < VanadielUniqueDay()
```

Comment:
on retail you can re-zone to get you once per day egg multiple times, so a localVar would be more appropriate

_I just gave away the secret to my egg factory, retail players. you're welcome._

Also its king of annoying when you do not want eggs, because that 1st swing of you pick/hatchet/sickle will be an egg and nothing else.

Reply:
come to think of it I think I mentioned this at some point in the past but then let it slide. Was this just moved from bigger logic block elsewhere? Github doesn't like large file diffs and I haven't dug all the way in yet.

Reply:
This seems a bit beyond the scope because it's not supposed to be a change in functionality. I think most servers would want this turned off, as it kind of ruins the event, in which case I'd need to make a setting.

Reply:
a "yes" would have sufficed bro.

I think that was my own reasoning too - getting eggs when you want darksteel or adaman or w/e sucks.